src/handler.ts

```diff
@@ -0,0 +1,46 @@
+import { URLExt } from '@jupyterlab/coreutils';
```

Comment:
This isn't being used yet because we are implementing our own handlers. I kept it in case we change the JS source in future to adopt this approach provided by JL4. If you want, I can remove it?

Reply:
I am find is you leave it just add a comment at the top describing it.

Reply:
Ok